test/test_pod_helpers.rb

```diff
@@ -112,4 +112,22 @@ def test_v
     assert_equal(1_001_000, v(1, 1, 0))
     assert_equal(1_001_001, v(1, 1, 1))
   end
+
+  def test_append_additional_build_settings
+    build_settings = { 'OTHER_LDFLAGS' => '-l"Pods-TestApp"' }
+
+    append_additional_build_settings!(build_settings, {})
+
+    assert_equal('-l"Pods-TestApp"', build_settings['OTHER_LDFLAGS'])
+
+    append_additional_build_settings!(build_settings, { 'OTHER_LDFLAGS' => ' -ObjC' })
```

Comment:
Can you also add a case where `OTHER_LDFLAGS` isn't already set in the destination object?

Reply:
Yeah, just spotted this bug, test case added